C/manual/ch_Reports.xml

```diff
@@ -3778,7 +3778,179 @@ Income (type INCOME)
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Equity &minus; General</glossterm>
+            <glossdef>
+              <para>The value shown for each account is its balance as of the start date of the selected budget,
+                modified by the projected changes in this budget. So if all goes according to the
+                budget, these balances would be the same as if you were to run a standard Balance
+                Sheet as of the end date of the budget period.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Equity &minus; Existing Retained Earnings</glossterm>
+            <glossdef>
+              <para>Retained Earnings as of the start date of the given budget. This is equivalent to the Retained
+                Earnings on the Balance Sheet when run for that start date.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Equity &minus; New Retained Earnings</glossterm>
+            <glossdef>
+              <para>The sum of budgeted Incomes minus the sum of budgeted Expenses. The profit predicted by this budget.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Equity &minus; Unrealized Losses</glossterm>
+            <glossdef>
+              <para>The total difference in value between all asset/liability accounts and their basis. This is
+                equivalent to the same total in the Balance Sheet as of the start date of the
+                budget.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Equity &minus; Existing Equity</glossterm>
+            <glossdef>
+              <para>The Total Equity from the Balance Sheet on the start date of the budget. It is comprised of the
+                balance of the selected Equity accounts, plus any existing Retained Earnings plus
+                Unrealized Gain/Losses.
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Equity &minus; New Equity</glossterm>
+            <glossdef>
+              <para>The sum of projected amounts for equity accounts in the budget added to New Retained Earnings
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Equity &minus; Total Equity</glossterm>
+            <glossdef>
+              <para>The sum of Existing Equity and New Equity
+              </para>
+            </glossdef>
+          </glossentry>
+
+          <glossentry>
+            <glossterm>Total Liabilities + Equity</glossterm>
+            <glossdef>
+              <para>Self Explanatory. This will equal Total Assets.
```

Comment:
… if the book is balanced.

Reply:
> … if the book is balanced.

Always. The Balance Sheet Report gets a book into balance by inserting a calculated Equity account, Unrealized Gains (or Losses depending on whether Equity is less or greater than Assets - Liabilities) to balance an out-of-balance book. Everything else in the report exactly reflects the balances in the accounts at the end of the selected date.